x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/helpers.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  ASSET_CRITICALITY_INDEX_PATTERN,
+  RISK_SCORE_INDEX_PATTERN,
+} from '../../../../common/constants';
+import type {
+  Entity,
+  UserEntity,
+} from '../../../../common/api/entity_analytics/entity_store/entities/common.gen';
+
+export const isUserEntity = (record: Entity): record is UserEntity =>
```

Comment:
i feel like we already have a bunch of these helpers around. Maybe we should add a tech debt ticket to move all of them to some common/shared folder

Reply:
I don't know why `isUserEntity ` is highlighted as new. This function was already there. I only added `sourceFieldToText`.

Reply:
Here is it in the main branch
https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/helpers.ts